pro/src/components/pages/Offerers/Offerer/VenueV1/fields/PricingPoint/PricingPoint.jsx

```diff
@@ -0,0 +1,120 @@
+import * as pcapi from 'repository/pcapi/pcapi'
+import { Field, useField } from 'react-final-form'
+import React, { useEffect, useState } from 'react'
+import Button from 'ui-kit/Button/Button'
+import classnames from 'classnames'
+import { dehumanizeId } from 'core/Venue/utils'
+import Icon from '../../../../../../layout/Icon'
+
+const PrincingPoint = ({ readOnly, offerer, venueId: venue }) => {
+  const [isButtonHidden, setIsButtonHidden] = useState(false)
+  const [isButtonDisabled, setIsButtonDisabled] = useState(true)
+  const [isInputDisabled, setIsInputDisabled] = useState(false)
+  const pricingPointSelectField = useField('venueSiret', {})
+
+  useEffect(() => {
+    setIsButtonDisabled(!pricingPointSelectField.input.value)
```

Comment:
Le useEffect est inutile ici

```suggestion
    const IsButtonDisabled = !pricingPointSelectField.input.value
```

aussi, j'utiliserait plutot quelquechose comme 'canSubmit' qui décrit fonctionnellement ce qui se passe.